backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/BiobankDirectoryLoader.java

```diff
@@ -16,8 +16,9 @@ public class BiobankDirectoryLoader extends ImportDataModelTask {
 
   private boolean staging = false;
 
-  public BiobankDirectoryLoader(Schema schema, Boolean includeDemoData) {
-    super(schema, includeDemoData);
+  public BiobankDirectoryLoader(
+      Database database, String schemaName, String description, Boolean includeDemoData) {
```

Comment:
Since it is a lot of the same arguments for all these function, it might be good to reduce the arguments by introducing a SchemaLoadingSettings class (or sth like that).